src/js/embed-code.js

```diff
@@ -4,23 +4,42 @@ var sprintf = require('sprintf');
 
 var uid = shortid.generate();
 
+
 // Public API
 // ----------
 
-var getIFrameEmbedCodeWithInlineResize = function(baseUrl) {
-  var id = getIFrameId();
-  return getEmbedCodeIFrame(id, baseUrl) + getEmbedCodeResizeScript(id);
-};
 
+//
+// All the three different functions for retrieving embed codes
+// have the same params
+//
+// baseUrl  -- baseUrl for the folder containing embed assets,
+//             like embed.js and resize.js. Normally also contains
+//             the index.html that bootstraps the embed
+//
+// embedUrl -- For multi-variant embeds, the same embed.js and resize.js
+//             is shared for multiple embeds. in this case embedUrl is different
+//             from baseUrl. This applies for example to all legacy lucify.com embeds
+//             In case the url to the embeds index.html is different from the baseUrl,
+//             this parameter should be used
+
+
+var getIFrameEmbedCodeWithInlineResize = function(baseUrl, embedUrl) {
+  var id = getIFrameId();
+  var embedUrl = getEmbedUrl(baseUrl, embedUrl);
```

Comment:
(This applies to all three similar functions.)

I'm generally not a fan of creating a local variable of the same name as a parameter. Makes reading the code more confusing and prone to bugs.